frontend/src/app/projects/project-detail/model-overview/model-overview.component.html

```diff
@@ -53,13 +53,15 @@ <h2>
             mat-mini-fab
```

Comment:
The model creation button should only be accessible when the user is manager. (It’s actually above but I cannot comment there).

Reply:
Good catch!